config/sync/core.extension.yml

```diff
@@ -35,7 +35,6 @@ module:
   page_cache: 0
   path: 0
   path_alias: 0
-  rsvplist: 0
```

Comment:
why is this and the other module getting deleted?

Reply:
I added them to gitignore because those were only for practice purposes, if i ever make any changes to them that should not be reflected on PR only helloworld module is of concern

Reply:
Should i remove them from gitignore?